package.json

```diff
@@ -32,6 +32,7 @@
     "graphql.macro": "^1.4.2",
     "hds-core": "^0.4.1",
     "hds-react": "^0.5.4",
+    "helsinki-utils": "https://github.com/City-of-Helsinki/helsinki-utils-js.git#0.1.0",
```

Comment:
Wonder if this can be added to devDependencies? Doesn't make a big difference, but I notice that it feels "safer" to update those..

Reply:
You are right 👍